src/app/layout.tsx

```
@@ -1,5 +1,5 @@
 import type { Metadata } from 'next';
-import './globals.css';
+import '../styles/globals.css';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)